src/Dockerfile

```
@@ -31,8 +31,8 @@ RUN apk add --no-cache git \
     less
 
 # download a the main repos from github
-RUN git clone --branch development-v6 https://github.com/pi-hole/AdminLTE.git /var/www/html/admin
-RUN git clone --branch development-v6 https://github.com/pi-hole/pi-hole.git /etc/.pihole
+RUN git clone --branch development-v6 https://github.com/pi-hole/AdminLTE.git /var/www/html/admin &&\
```

```suggestion
RUN git clone --branch development-v6 https://github.com/pi-hole/AdminLTE.git /var/www/html/admin && \
```

Please, add a space between `&&` and `\` for readability (and consistence with other multi-line commands).